lib/zuora/soap/calls/subscribe.rb

```diff
@@ -0,0 +1,85 @@
+module Zuora
+  module Soap
+    module Calls
+      module Subscribe
+        def self.xml_builder
+          # account, batch, payment_method, bill_to_contact, subscription
```

Comment:
Yep, all this is WIP, doesn't break the specs but sets up the new structures. In the subsequent PR #30 these details get filled in.